microcosm_daemon/runner.py

```diff
@@ -0,0 +1,48 @@
+"""
+Execution abstraction.
```

Comment:
Defining a "runner" layer keeps the multiprocessing support abstracted.

The only catch: there's some trickiness to ensuring that references passed to other processes work. It seems fine to pass object instances (e.g. of `Daemon`), but not to pass function references on those instances (e.g. `Daemon.start`) probably because for some reason related to memory addresses.

I erred on the side of forking to different processes as early as possible, before object graph construction.